src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
+          <img src="touxiang.jpg" alt="picture" style="text-align:center">
+        </div>
+
+        <div>
+          <div id="Slider">
+            <!--Slider About me-->
+            <div id="AboutMe_Container">
+              <div id="AboutMe_title">
+                <h2 style="text-transform:uppercase">About Me</h2>
+              </div>
+              <div id=AboutMe_Icon>
```

Comment:
This is no strict naming convesion rules of id attribute, but we prefer something like "about-me-container". This is also suggested by google: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters